llama.cpp

```diff
@@ -271,10 +271,11 @@ static std::string llama_format_tensor_shape(const std::vector<uint32_t> & ne) {
 
 static const char * llama_format_type(enum ggml_type type) {
     switch (type) {
-        case GGML_TYPE_F32: return "f32";
-        case GGML_TYPE_F16: return "f16";
-        case GGML_TYPE_Q4_0: return "q4_0";
-        case GGML_TYPE_Q4_1: return "q4_1";
+        case GGML_TYPE_F32:
```

Comment:
Maybe we can drop `llama_format_type` completely and just use `ggml_type_name` instead?
There is very little added in checking the type against these 4 options explicitly, since it is only used in debug prints.